OrphanedAssets/Editor/MaterialDependencies.cs

```diff
@@ -0,0 +1,328 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using UnityEditor;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+class MaterialDependencies : EditorWindow
+{
+	static readonly string[] k_SearchFolders = { "Assets/EditorVR/Workspaces" };
+	static readonly string[] k_ExcludePaths = { "libs" };
+
+	[MenuItem("Window/Material Dependencies")]
+	static void Init()
+	{
+		var window = GetWindow<MaterialDependencies>();
+		//var position = window.position;
```

Comment:
any reason to keep the commented code?

Reply:
Not sure. I ended up losing the window at one point because of my monitor setup, but it might be a bug that's since been fixed in Unity.

Can probably delete it, I just didn't notice it when I put this in the repo